signl4/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "SIGNL4",
+  "maintainer": "success@signl4.com",
+  "manifest_version": "1.0.0",
+  "name": "signl4",
+  "metric_prefix": "signl4.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Get notified of your Datadog alerts and take actions using SIGNL4.",
+  "guid": "02b6d170-8b2e-4de4-b3a9-afb67183cb5e",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "SIGNL4 Integration for Datadog",
```

Comment:
This is the name that is displayed in our app and our integrations docs/corporate site. I suggest changing this to just `SIGNL4`

Reply:
Thanks a lot @scseanchow.